makeBuild.py

```diff
@@ -99,7 +99,7 @@ def _fix_ids_for_html4(tree):
             #continue
 
         # rest api book is a pain and doesn't convert well
-        if book == "rest_api":
+        if book == "rest_api" or book == "microshift_rest_api":
```

Comment:
@eggfoobar, if you do this, the files can't be rebuilt to be published on the portal. (So you can't do this.)

Reply:
Thanks @kalexand-rh , I removed it and did some more work to try and fix the xrefs, not sure if that's too large. Do you have a way to build a temp portal site like we do with netlify now?